network/network_test.go

```diff
@@ -21,7 +22,7 @@ func makeTestRelay(t *testing.T) host.Host {
 	t.Helper()
 
 	h, err := lp2p.New(
-		lp2p.ListenAddrStrings("/ip4/0.0.0.0/tcp/0"),
+		lp2p.ListenAddrStrings("/ip4/127.0.0.1/tcp/0"),
```

Comment:
I think if we use `127.0.0.1` for tests it perfectly works on shell pipeline runners but in docker pipeline runners it would be better to use `0.0.0.0` which work's like `127.0.0.1` in shell runners.

Reply:
As I tested, in some case we had connection failure (probably firewall?) with `0.0.0.0`, but `127.0.01` for ipv4 and `::1` for ipv6 works fine.

Reply:
Consider, the ports in these  tests are not going to expose. Like running tests inside docker.